griddedio/GriddedIO.F90

```diff
@@ -254,6 +257,35 @@ subroutine set_default_chunking(this,rc)
 
      end subroutine set_default_chunking
 
+     subroutine check_chunking(this,lev_size,rc)
+        class (MAPL_GriddedIO), intent(inout) :: this
+        integer, intent(in) :: lev_size
+        integer, optional, intent(out) :: rc
+
+        integer ::  global_dim(3)
+        integer :: status
+
+        call MAPL_GridGet(this%output_grid,globalCellCountPerDim=global_dim,rc=status)
+        _VERIFY(status)
+        if (global_dim(1)*6 == global_dim(2)) then
+           _ASSERT(this%chunking(1) <= global_dim(1), "Bad chunk size")
```

Comment:
Should we have more "explicit" asserts here? Like "chunk for X dim, this%chunking(1), must be less than or equal to global_dim(1)"? Not sure.

Reply:
Added informative messages